README.md

```diff
@@ -29,7 +29,10 @@ if __name__ == '__main__':
         photos_picker = PhotosPicker(picker, (), uploader)
         photos_picker.run()
     except Exception as err:
-        print err.message    
+        if err.message:
+            print err.message
```

Comment:
No brackets like the second example?

Reply:
No :) In the second example (more complex) I need the "future" version of print. This version allows me to remove the trailing "\n" and thus to update the displayed progress percentage rate.

I don't need that in the first example, so I use old print version.

Thanks for reviewing :)